app/allocation/steps/person-assign.js

```diff
@@ -0,0 +1,106 @@
+const { create } = require('../../move/steps')
+const controllers = require('../controllers/person-assign')
+
+const personAssign = { ...create }
+
+const updateAssignStep = (key, props) => {
+  personAssign[key] = { ...personAssign[key], ...props }
+}
+
+updateAssignStep('/person-lookup-prison-number', {
+  controller: controllers.PersonSearch,
+  next: 'person-lookup-results',
+})
+
+updateAssignStep('/person-lookup-results', {
+  controller: controllers.PersonSearchResults,
+  next: 'agreement-status',
+})
+
+updateAssignStep('/agreement-status', {
+  next: 'release-status',
+})
+
+updateAssignStep('/release-status', {
+  controller: controllers.Assessment,
+  next: 'special-vehicle',
+})
+
+updateAssignStep('/special-vehicle', {
+  controller: controllers.Assessment,
+  pageTitle: 'moves::steps.health_information.heading',
+  fields: ['special_vehicle_check'],
+  next: [
+    {
+      field: 'special_vehicle_check',
+      value: 'true',
+      next: 'no-special-vehicle',
+    },
+    'save',
+  ],
+})
+
+updateAssignStep('/no-special-vehicle', {
+  templatePath: 'allocation/views/person-assign/',
+  template: 'no-special-vehicle',
+  pageTitle: 'moves::steps.person_search.heading',
+})
+
+updateAssignStep('/save', {
+  controller: controllers.Save,
+  next: 'confirmation',
+})
+
+updateAssignStep('/confirmation', {
+  checkJourney: false,
+  checkSession: false,
+  controller: controllers.Confirmation,
+  templatePath: 'allocation/views/person-assign/',
+  template: 'confirmation',
+})
+
+module.exports = personAssign
+
+// const personAssignSteps = {
```

Comment:
I actually think this version is easier to read and understand. The order of the steps is important and it would be better to keep this structure  for readability I think.

Reply:
I agree - but it wasn't working. So I did it this way to at least be able to show it working. Looking to redo it now.